### PR TITLE
#10 Create welcome section component

### LINKED ISSUE #10: Create welcome section component
1. Create page description
2. Create registration button (should redirect to registration page)
3. Add gorgeous image))

### PR DESCRIPTION
**Pull request by nikolakozaryan**
# Description

1. Fix webpack config (add fonts loader).
2. Add common project styles.
3. Fix Button component style.
4. Create Main -> Welcome component. 

Fixes #10 

## Type of change

Please choose suitable options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Screenshot is attached or not needed

Screenshot: 
image

### CODE REVIEW

**Comment by khvalkoilya** [comment]
Could you please add screenshot?

File: src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Main from './components/pages/Main/Main';
+import "./common/scss/main.scss"
+
+const App = () => {
```
Comment (khvalkoilya) [inline src/App.tsx]: Do not forget to add TypeScript

File: src/components/pages/Main/Main.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Welcome from './components/Welcome/Welcome';
+
+const Main = () => {
```
Comment (khvalkoilya) [inline src/components/pages/Main/Main.tsx]: The same here

File: src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  return (
+    <div>
```
Comment (khvalkoilya) [inline src/App.tsx]: Do we really need this div?
In future you can use fragment as a parent tag (<> </>)

File: src/components/pages/Main/components/Welcome/Welcome.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './Welcome.module.scss'
+import Button from '../../../../ui/button/Button';
+
+const Welcome = () => {
```
Comment (khvalkoilya) [inline src/components/pages/Main/components/Welcome/Welcome.tsx]: if the component only returns something, then it is possible without the word 'return'

File: src/common/scss/base/_base.scss [hunk 5]
```diff
@@ -0,0 +1,9 @@
+* {
+    box-sizing: border-box;
+    color: $font-color;
+}
+
+#root {
+    width: 1440px;
```
Comment (khvalkoilya) [inline src/common/scss/base/_base.scss]: I guess it should be max-width. What do you think?

File: src/App.tsx (hunk 1 above)
Comment (nikolakozaryan) [reply src/App.tsx]: Done

File: src/components/pages/Main/Main.tsx (hunk 2 above)
Comment (nikolakozaryan) [reply src/components/pages/Main/Main.tsx]: Done

File: src/components/pages/Main/components/Welcome/Welcome.tsx (hunk 4 above)
Comment (nikolakozaryan) [reply src/components/pages/Main/components/Welcome/Welcome.tsx]: Done

File: src/common/scss/base/_base.scss (hunk 5 above)
Comment (nikolakozaryan) [reply src/common/scss/base/_base.scss]: Agree with you. Changed to max-width.

File: src/App.tsx (hunk 3 above)
Comment (nikolakozaryan) [reply src/App.tsx]: Done

**Review by khvalkoilya — APPROVED** [review]
Please wait someone else approve before merge

**Review by YaGrig — APPROVED** [review]

**Pull request merged**